src/components/progress-report/workflow/progress-report-workflow.edgedb.repository.ts

```diff
@@ -57,37 +57,39 @@ export class ProgressReportWorkflowEdgeDBRepository
   }
 
   async getProjectMemberInfoByReportId(reportId: ID) {
-    const progressReport = e.cast(e.ProgressReport, e.uuid(reportId));
-    const query = e.select(
-      e.select(progressReport).project.members.user,
-      (user) => ({
-        id: true,
-        email2: e.assert_exists(user.email),
-        roles: true,
-      }),
-    );
-    const results = await this.db.run(query);
-    return results.map(({ id, email2, roles }) => ({
-      id,
-      email: email2,
-      roles,
-    }));
+    const query = edgeql(`
+      with report := <ProgressReport><uuid>$reportId,
+      users := (select report.project.members.user filter exists .email)
+      select users { id, email := assert_exists(.email), roles }
+    `);
+    return await this.db.run(query, { reportId });
   }
 
   async getUserIdByEmails(emails: readonly string[]) {
-    const query = e.params({ emails: e.array(e.str) }, ({ emails }) =>
-      e.select(e.User, (user) => ({
-        filter: e.op(user.email, 'in', e.array_unpack(emails)),
-        id: true,
-        email2: e.assert_exists(user.email),
-      })),
-    );
-    const results = await this.db.run(query, { emails });
-    return results.map(({ id, email2 }) => ({
-      id,
-      email: email2,
-    }));
+    const query = edgeql(`
+      with emails := array_unpack(<array<str>>$emails),
+      users := (select User filter .email in emails)
+      select users {
+        id,
+        email := assert_exists(.email),
+      } 
+    `);
+    return await this.db.run(query, { emails });
   }
+  // async getUserIdByEmails(emails: readonly string[]) {
```

Comment:
Probably just forgot to delete this?

Reply:
oops, forgot to save the file after deleting it.  will push another commit now